mgwr/gwr.py

```diff
@@ -86,6 +87,10 @@ class GWR(GLM):
     name_x        : list of strings
                     Names of independent variables for use in output
 
+    n_jobs        : integer
+                    The number of jobs (default 1) to run in parallel. -1 means using all processors.
```

Comment:
I know I am coming late to the party but would you consider using -1 as a default? That is quite common across ML world and it is what users generally expect.

Reply:
i was curious what others' opinion was on this... i tend to default to -1 personally, but joblib itself [indirectly] defaults to 1, and in esda [we do both](https://github.com/search?q=repo%3Apysal%2Fesda%20path%3A%2F%5Eesda%5C%2F%2F%20n_jobs&type=code) (join counts are conservative, G defaults to -1, etc)

Reply:
i think [scikit usually defaults to 1](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.RandomizedSearchCV.html) so i dont think there's a standard expectation in ML world

Reply:
I don't mind either as long as it is documented (which it is). But for heavily parallelisable code like this one, I tend to prefer parallel execution by default.